### PR TITLE
Support SRv6 

### PR DESCRIPTION
**Pull request by zvfvrv**
This PR is related to the Issue #141

### CODE REVIEW

**Comment by AloysAugustin** [comment]
Thanks @zvfvrv , this looks good :)
From a first look I was curious about the inferLocalSidAddr function, this looks like a bit of a hack for now, could you give a bit more info about potential ways to make this work in any cluster?

**Comment by zvfvrv** [comment]
Hi @AloysAugustin, yes inferLocalSidAdd is a dev function that actually I will remove soon :) . The idea is to provide through the resources IPPool and ConfigMap the collection of IP addresses to be used by the agent, in this way it should be completely configurable and should it work in any cluster.

**Comment by jjgraham** [comment]
When do you think SRv6 support will be available in your containers ???

**Comment by jjgraham** [comment]
So close !

File: calico-vpp-agent/Makefile [hunk 1]
```diff
@@ -26,6 +26,7 @@ image: build gobgp
 	@if [ "${ALSO_LATEST}" = "y" ]; then \
 		docker tag calicovpp/agent:$(TAG) calicovpp/agent:latest; \
 	fi
+	docker image prune -f
```
Comment (sknat) [inline calico-vpp-agent/Makefile:29]: Does this have to be part of the `make image` ? 
Because it might impact non related images present on the machine

**Comment by sknat** [comment]
Appart from a minor comment, lgtm 😄

There might be incoming conflicts with #283 but as this PR has been open for long, we can merge it first, and rebase #283 on top.

File: calico-vpp-agent/Makefile (hunk 1 above)
Comment (zvfvrv) [reply calico-vpp-agent/Makefile]: Yes I can remove it

Comment (sknat) [reply calico-vpp-agent/Makefile]: thanks 😄

**Review by sknat — APPROVED** [review]

**Pull request merged**